tests/cypress/support/commands.ts

```diff
@@ -75,6 +100,19 @@ Cypress.Commands.add('nameSpaceMenuToggle', (namespaceName) => {
   cy.get('.icon.icon-chevron-up').click({ force: true });
 })
 
+// Go to specific Continuous Delivery Sub Menu
+Cypress.Commands.add('accesMenuSelection', (firstAccessMenu='Continuous Delivery',secondAccessMenu) => {
```

Comment:
can this command renamed, something like: `accessContinuoueDeliverySubmenu` ? as First Menu is default arg...

Reply:
We could, but I intentionally left it as this, because it can be used with other options aside from Continuos delivery.
Actually, I should probably rephrase the comment better if you agree.

Reply:
yes, please as it is confusing bit :smile: